agave_app/python/pyrenderer.cpp

```diff
@@ -196,12 +196,20 @@ OffscreenRenderer::LoadOmeTif(const std::string& s)
 }
 // load a volume
 int
-OffscreenRenderer::LoadVolumeFromFile(const std::string& s, int time, int scene)
+OffscreenRenderer::LoadVolumeFromFile(const std::string& s, int scene, int time)
```

Comment:
Changing the ordering of a pair of like-typed args is always worrisome- can create hard to find bugs.

Reply:
or in this case, fix them 😟